lib/plugins/sms/twilio/twilio.js

```diff
@@ -14,7 +14,8 @@ function sendMessage (account, rec) {
       const client = twilio(account.accountSid, account.authToken)
       const body = rec.sms.body
       const _toNumber = rec.sms.toNumber || account.toNumber
-      const from = _.startsWith('+')(account.fromNumber)
+      const from = (_.startsWith('+')(account.fromNumber) 
+        || isNaN(String(account.fromNumber).replace(/\s/g,'')))
```

Comment:
Prefer ``_.isNumber``, since ``isNaN`` isn't really meant for this and has some weird special cases.

Reply:
Fixed!